webhook_features/src/features/summary_comment.rs

```diff
@@ -294,9 +314,9 @@ lazy_static! {
     static ref ACK_PATTERNS: Vec<(Regex, AckType)> = vec![
         (r"\b(Approach ACK)\b", AckType::ApproachAck),
         (r"\b(Approach NACK)\b", AckType::ApproachNack),
-        (r"\b(NACK)\b", AckType::ConceptNack),
         (r"\b(Concept ACK)\b", AckType::ConceptAck),
         (r"(ACK)(?:\s*)([0-9a-f]{6,40})\b", AckType::Ack),
+        (r"\b(NACK)\b", AckType::ConceptNack),
```

Comment:
Won't this cause `NACK ffaabbccdd11` to be parsed as ACK?

Reply:
Right, dropped aa1ceb46d2eb4269ae68396db56814290f7587ce.
Maybe we could add a check in the `ACK <hash>` pattern that checks that the word doesn't start with an N. I think this requires lookaheads and the standard regex crate of rust doesn't support them.

Reply:
unrelated: Could add a test for this?

Reply:
Done